sentimentVisualizer/visualize.html

```diff
@@ -93,8 +100,20 @@
                         <form class="navbar-form hidden-xs ng-pristine ng-valid">
                             <div class="form-group">
                                 <div class="input-group input-group-xs ">
+<<<<<<< HEAD
+                                    <input name="q" type="text" id="query"
+                                           class="form-control" ng-model="query"
+                                           title="Search Twitter and the loklak network"
+                                           placeholder="Search for a tweet text">
+                                    <span class="input-group-btn">
+                                    <button class="btn btn btn-primary" href='#'
+                                            type="button" id="button"
+                                            ng-click="Search()">Visualize Emotion</button>
+                                    </span>
+=======
                                     <input name="q" type="text" id="query" class="form-control" ng-model="query" title="Search Twitter and the loklak network" placeholder="Search for a tweet text">
-                                    <span class="input-group-btn"><button class="btn btn btn-primary" href='#' type="button" id="button" ng-click="Search()">Visualize Emotion</button></span>
+                                    <span class="input-group-btn"><button class="btn btn btn-primary" href="#" type="button" id="button" ng-click="Search()">Visualize Emotion</button></span>
```

Comment:
Line is longer than allowed. (190 > 129)

*LineLengthBear, severity NORMAL, section `default`.*